x/multisig/handlers.go

```diff
@@ -0,0 +1,174 @@
+package multisig
+
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/orm"
+	"github.com/iov-one/weave/x"
+)
+
+const (
+	creationCost int64 = 1
```

Comment:
let's raise these number a bit... should be say 3x as expensive as SendMsg
I know it is arbitrary now, but good to have some meaning captured there